### ISSUE TITLE
ask for help

### ISSUE THREAD

**Issue by 22wei22** [ricvolpi/adversarial_feature_augmentation#5]
when I train the generator of step 1. I find that the discriminator is powerful .the g_loss will be 1 and the d_loss will be 0. How could I do

**Comment by ricvolpi** [comment]
Did you observe this behavior running Step 0 and Step 1 out of the box, or did you modify something?

**Comment by 22wei22** [comment]
Yeah I implement it in pytorch. Now I can not converge it.  In step 0 ,the accuracy of  pretrained model is 0.87. Is it right？

**Comment by ricvolpi** [comment]
No, the accuracy of the model trained at Step 0 should be much lower (check the paper - Table 2, first row).
